include/circt/Dialect/Seq/SeqOps.h

```diff
@@ -13,6 +13,7 @@
 #ifndef CIRCT_DIALECT_SEQ_SEQOPS_H
 #define CIRCT_DIALECT_SEQ_SEQOPS_H
 
+#include "mlir/Bytecode/BytecodeOpInterface.h"
```

Comment:
Why is this required? Odd.

Reply:
Odd indeed.  I don't know, seems like a bug to be honest, maybe they're missing a header themselves.